infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +124,34 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def is_crash_valid(self, test_case):
+    """Checks if a crash is reproducible, and if it is, whether it's a new
+    regression that cannot be reproduced with the latest OSS-Fuzz build.
+
+    NOTE: If no old_build_target is specified the crash is assumed introduced
+    by the pull request if it is reproducible.
+
+    Args:
+      test_case: The path to the test_case that triggered the crash.
+
+    Returns:
+      True if the crash was introduced by the current pull request.
+    """
+    exists_in_pr = self.is_reproducible(test_case,
+                                        os.path.dirname(self.target_path))
+    if not self.ossfuzz_build_path:
+      return exists_in_pr
+
+    if not exists_in_pr:
+      logging.info('Crash is not reproducible.')
+      return False
+    exists_in_ossfuzz = self.is_reproducible(test_case, self.ossfuzz_build_path)
+    if exists_in_pr and not exists_in_ossfuzz:
```

Comment:
I don't like the fact that we download the old build no matter what. I think it would be better to only do it if needed, maybe in a helper method that checks if the bug is not_novel here.
@Dor1s WDYT?

Reply:
definitely